src/rendering/batcher/shared/Batcher.ts

```diff
@@ -107,24 +135,23 @@ let BATCH_TICK = 0;
  */
 export interface BatcherOptions
 {
-    /** The size of the vertex buffer. */
-    vertexSize?: number;
-    /** The size of the index buffer. */
-    indexSize?: number;
     /** The maximum number of textures per batch. */
     maxTextures?: number;
+    vertexInitialSize?: number;
```

Comment:
do we need to change the name here? Seems like a avoidable breaking change

Reply:
moving terminology from vertex to attribute for consistency, i think this is ok. (no more breaking than the other changes iyswi?)